sharekit/sharekit/RequestData.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace sharekit
+{
+    public class RequestData
+    {
+#pragma warning disable IDE1006
```

Comment:
Since you're using `Newtonsoft` for the JSON serialization, I recommend naming these according to the C# best practices instead of using the #pragma and using [SerializationAttributes](https://www.newtonsoft.com/json/help/html/SerializationAttributes.htm) on the properties.